Source/WebKit/UIProcess/WebAuthentication/Cocoa/AuthenticationServicesForwardDeclarations.h

```diff
@@ -359,6 +359,59 @@ typedef NSString *ASAuthorizationPublicKeyCredentialResidentKeyPreference;
 
 @end
 
+
+typedef NS_ENUM(NSInteger, ASAuthorizationPlatformPublicKeyCredentialRegistrationRequestStyle) {
+    ASAuthorizationPlatformPublicKeyCredentialRegistrationRequestStyleStandard,
+    ASAuthorizationPlatformPublicKeyCredentialRegistrationRequestStyleConditional,
+};
+
+@interface ASAuthorizationPublicKeyCredentialPRFAssertionOutput: NSObject
```

Comment:
```suggestion
@interface ASAuthorizationPublicKeyCredentialPRFAssertionOutput : NSObject
```

Many instances throughout this file.